.github/workflows/build.yml

```
@@ -463,6 +463,69 @@ jobs:
           name: 'PackSquash CLI AppImage zsync (arm64)'
           path: PackSquash-*-aarch64.AppImage.zsync
 
+  docker-image:
+
+    runs-on: ubuntu-latest
+
+    env:
+      # Use docker.io for Docker Hub if empty
+      REGISTRY: ghcr.io
+      # github.repository as <account>/<repo>
+      IMAGE_NAME: ${{ github.repository }}
+
+    needs: [ linux-x64-gnu, linux-aarch64-gnu ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build Docker image metadata
+        id: dockermeta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          # generate Docker tags: edge for every commit on master, semver for every semantic tag
+          tags: |
+            type=edge,branch=master
+            type=semver,pattern={{version}}
+        
+      - name: Download PackSquash X86-64 AppImage
+        uses: actions/download-artifact@v3.0.0
+        with:
+          name: 'PackSquash CLI AppImage (x64)'
+
+      - name: Download PackSquash AArch64-ARM64 AppImage
+        uses: actions/download-artifact@v3.0.0
+        with:
+          name: 'PackSquash CLI AppImage (arm64)'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          version: "v0.5.1"
+          buildkitd-flags: --debug
+
+      # We need a github token with write:package permission in reposiotry secret for this to work
+      - name: Login to ghcr.io
+        uses: docker/login-action@v1
```

```suggestion
      - name: Login to ghcr.io
        if: github.event_name != 'pull_request'
        uses: docker/login-action@v2
```

It doesn't need to log in and should use v2.
https://github.com/docker/build-push-action/blob/master/docs/advanced/tags-labels.md

---

Resolved in https://github.com/ComunidadAylas/PackSquash/pull/111/commits/0d7bc6680135eb3c23e7628235625b29bfd52d5b